Article/Article.js

```diff
@@ -2,28 +2,34 @@
 
 class Article {
   constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    // assign this.domElement to the passed in domElement [DONE]
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class. [DONE]
+    this.expandButton = domElement.querySelector('.expandButton');
+    console.log('Creating Expand Button reference', domElement, this.expandButton);
+    // Using your expandButton reference, update the text on your expandButton to say "expand" [DONE]
+    this.expandButton.textContent = 'Expand';
+    // Set a click handler on the expandButton reference, calling the expandArticle method. [DONE]
+    this.expandButton.addEventListener('click', () => this.expandArticle())
   }
 
   expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
+    // Using our reference to the domElement, toggle a class to expand or hide the article. [DONE]
+    this.domElement.classList.toggle('article-open')
 
   }
 }
 
 /* START HERE: 
 
-- Select all classes named ".article" and assign that value to the articles variable.  
+- Select all classes named ".article" and assign that value to the articles variable.  [DONE]
 
-- With your selection in place, now chain .forEach() on to the articles variable to iterate over the articles NodeList and create a new instance of Article by passing in each article as a parameter to the Article class.
+- With your selection in place, now chain .forEach() on to the articles variable to iterate over the articles NodeList and create a new instance of Article by passing in each article as a parameter to the Article class. [DONE]
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll('.article');
+
+articles.forEach(article => {
+  const articleInstance = new Article(article)
+});
```

Comment:
One way to clean this up a tad
`document.querySelectorAll(".article").forEach(article => new Article(article));`